src/files/FilesPage.js

```diff
@@ -214,12 +230,17 @@ class FilesPage extends React.Component {
               <Breadcrumbs path={files.path} onClick={doFilesNavigateTo} />
 
               { files.type === 'directory'
-                ? <FileInput
-                  className='ml-auto'
-                  onMakeDir={this.makeDir}
-                  onAddFiles={this.add}
-                  onAddByPath={this.addByPath}
-                  addProgress={writeFilesProgress} />
+                ? <div className='ml-auto flex items-center'>
+                  <span
```

Comment:
This would be better as a `<Button>` with a `bg='bg-transparent'`, so that we get hover styles and focus. 

We need to improve our a11y reporting too, as this'd be difficult for assistive technologies to spot as clickable.

Reply:
Done. Although we'll need to refactor the `<Button>` component, as some classes end up not being overridden, e.g. the `fw` one!